elastic/kibana
# [Screenshotting] Have Darwin download Chromium at startup, add ARM support for Darwin

## Summary

Closes https://github.com/elastic/kibana/issues/109949
Closes https://github.com/elastic/kibana/issues/122092
Closes https://github.com/elastic/kibana/issues/109545

1. Updates the Chromium browser for the darwin-x64 platform from 856583 to 901912.
2. Adds support for PDF/PNG reports in darwin-arm64 using Chromium revision 901913.
3. Chromium is downloaded the first time Kibana server starts up, for both Darwin architectures only.

### Checklist

- [x] Ensure that Darwin distributables are not bundled with ANY headless browser

### Other changes

### For maintainers


### To Test
1. Run `yarn build --skip-os-packages` in this branch
2. Ensure that the darwin artifacts do not contain a Chromium binary
3. Run the Darwin builds on Mac x64 and Mac arm64
4. Ensure that Chromium is downloaded at server startup
5. Ensure that Chromium is NOT re-downloaded on subsequent server startup trials

### Release note
1. Adds support for PNG and PDF reports on Darwin Arm64 architecture.
2. Chromium binaries used for PNG and PDF reports are no longer bundled with the Darwin-based distributables. The binaries will be downloaded into the Kibana installation folder the first time Kibana is started.

### Documentation section added
![image](https://user-images.githubusercontent.com/908371/148322803-01a78c62-69da-4338-84f9-13e48383f519.png)
